lib/form/validate.js

```diff
@@ -206,7 +232,7 @@ const validate = (raw, type) => {
     settings.all = allSettings[value.plugin];
 
     // Run the validation function
-    result.validation = plugin.validation[plugin.inputs[result.input].validation.function](input, settings);
+    result.validation = required(input, settings, plugin.validation[plugin.inputs[result.input].validation.function]);
```

Comment:
Eek. That's one too many things. I know it's more lines, but can we not have `required` run `validation`? Can we run the `required` check _first_ and, if it's not required, _then_ run `validation` in here?